spec/features/rooms/index_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "The Room", type: :feature do
+  before(:each) do
+    test_data
+  end
+  describe "#index page" do
+    it "displays all rooms, their rates/suites, the hotel and the number of guests per room" do
+      @hank.rooms.push(@hyatt_room_3)
+      @jane.rooms.push(@hyatt_room_3)
+      @barbie.rooms.push(@hyatt_room_3)
+
+      @barbie.rooms.push(@daysinn_room_6)
+      @billy.rooms.push(@daysinn_room_6)
+
+      @hank.rooms.push(@daysinn_room_5)
+
+      visit "/rooms"
+
+      expect(page).to have_content(@motel_8_room_1.hotel.name)
+      expect(page).to have_content(@motel_8_room_1.rate)
+      expect(page).to have_content(@motel_8_room_1.suite)
+
+      expect(page).to have_content(@hyatt_room_3.hotel.name)
+      expect(page).to have_content(@hyatt_room_3.rate)
+      expect(page).to have_content(@hyatt_room_3.suite)
+      expect(page).to have_content(@hyatt_room_3.guest_count)
```

Comment:
Same note here. Once this variable resolves, you're just asserting that a small integer appears on the page. Making these assertions more explicit with fixed text and within blocks would make the test more meaningful